src/PresentationalComponents/Export/TablePage.js

```diff
@@ -16,7 +16,7 @@ import {
   Thead,
   Tr,
 } from '@patternfly/react-table';
-import global_BackgroundColor_200 from '@patternfly/react-tokens/dist/js/global_BackgroundColor_200';
+import global_palette_black_200 from '@patternfly/react-tokens';
```

Comment:
I doubt this is correct. The default export of '@patternfly/react-tokens' is for sure not the object for `global_palette_black_200` :)

Reply:
Oh yes, I will change these values soon. Working on the other advisor related issues.